LibreNMS/Authentication/AuthorizerBase.php

```diff
@@ -78,6 +78,11 @@ public function getExternalUsername()
             return $_SERVER['REMOTE_USER'];
```

Comment:
```php
return $_SERVER[Config::has('http_auth_header', 'REMOTE_USER')] ?? $_SERVER['PHP_AUTH_USER'];
```

Reply:
Default value handling and i18n string is added